src/hooks/core/chat/useDMMessage.ts

```diff
@@ -80,12 +74,13 @@ const useDMMessage = () => {
   const sendMessageDM = async (
     id,
     source: 'post' | 'comment',
-    channelCategory: ChannelCategory
+    channelCategory: ChannelCategory,
+    feedId: string
   ) => {
     const initChat =
       channelCategory === 'ANONYMOUS'
-        ? await initChatFromPostAnon({source, id})
-        : await initChatFromPost({source, id});
+        ? await initChatFromPostAnon({source, id, feedId})
+        : await initChatFromPost({source, id, postId: feedId});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `feedId` as a parameter to `sendMessageDM` and its subsequent usage in `initChatFromPostAnon` and `initChatFromPost` calls is a significant change. Ensure that all places where `sendMessageDM` is called have been updated to pass this new argument. Also, make sure that the functions `initChatFromPostAnon` and `initChatFromPost` handle this new parameter correctly.
```diff
-    channelCategory: ChannelCategory
+    channelCategory: ChannelCategory,
+    feedId: string
```
```diff
-        ? await initChatFromPostAnon({source, id})
-        : await initChatFromPost({source, id});
+        ? await initChatFromPostAnon({source, id, feedId})
+        : await initChatFromPost({source, id, postId: feedId});
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->